Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.23.5-alpine3.20 as builder
+RUN apk add --no-cache git gmp-dev build-base g++ openssl-dev
+WORKDIR /app
+COPY go.mod go.sum ./
+RUN go mod download
+COPY . .
+RUN CGO_ENABLED=0 go build -ldflags "-s -w" -trimpath -o ./build/ezex-users ./cmd/server.go
```

Comment:
You can replace it with: `RUN make release`

The Docekrfile should be simplified. Look as an example to [Dockerfiel](https://github.com/ezex-io/ezex-notification/blob/main/Dockerfile) of the notification service.